cmd/autoscaler/main.go

```diff
@@ -231,3 +265,39 @@ func flush(logger *zap.SugaredLogger) {
 	logger.Sync()
 	metrics.FlushExporter()
 }
+
+func forwardingStatsNetworking(bucketSize int, accept func(sm asmetrics.StatMessage),
```

Comment:
let's have a named type for `func(asmetrics.StatMessage)` it is quite prevalent.

Reply:
done